app/docker/views/containers/containers.html

```diff
@@ -6,7 +6,14 @@
   </rd-header-title>
   <rd-header-content>Containers</rd-header-content>
 </rd-header>
-
+<information-panel title-text="Offline mode" cant-dismiss="true" ng-if="endpointStatus !== 1">
```

Comment:
Please remove any `cant-dismiss` attribute in other views as well (see my comment regarding information-panel.js).